roo_on_rails.gemspec

```diff
@@ -30,7 +30,13 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
+  spec.add_runtime_dependency 'dotenv-rails', '~> 2.1'
+  spec.add_runtime_dependency 'newrelic_rpm', '~> 3.17'
+  spec.add_runtime_dependency 'rails', '~> 5.0'
+
   spec.add_development_dependency "bundler", "~> 1.13"
   spec.add_development_dependency "rake", "~> 10.0"
   spec.add_development_dependency "rspec", "~> 3.0"
+  spec.add_development_dependency 'thor', '~> 0.19'
+  spec.add_development_dependency 'pry'
```

Comment:
well, yes, you can print documentation and source of methods for example.
But you can also `cd` into objects and classes and `ls` their instance methods, variables and ancestors, and move down and up as if the object-oriented hierarchy was a Unix file system.

Here there are more examples: http://pryrepl.org/